apps/server/src/shared/domain/entity/task.entity.ts

```diff
@@ -90,6 +91,11 @@ export class Task extends BaseEntityWithTimestamps implements ILearnroomElement,
 	@ManyToMany('User', undefined, { fieldName: 'archived' })
 	finished = new Collection<User>(this);
 
+	// The fileIds property is used for syncing the new filerecords collection with the old files collection.
+	// It can be removed after transitioning file-handling to the new files-storage-microservice is completed.
+	@Property({ nullable: true })
+	fileIds?: ObjectId[];
```

Comment:
Ahh we do not have fileIds for on this place for now?
Then we can use mongoose to load old references. With this we can avoid to use file entity.